FieldworkAIDriver.lua

```diff
@@ -971,6 +971,11 @@ function FieldworkAIDriver:lowerImplements()
 		implement.object:aiImplementStartLine()
 	end
 	self.vehicle:raiseStateChange(Vehicle.STATE_CHANGE_AI_START_LINE)
+	
+	if g_server == nil then 
```

Comment:
shouldn't this be at the very beginning of the function? Aren't those start line events propagated anyway?

Reply:
the problem is FieldworkAIDriver:startWork() calls -> FieldworkAIDriver:lowerImplements() and there gets called self.ppc:isReversing()

solution might be to have the Start/Stop Event only call:
-self.vehicle:raiseAIEvent("onAIStart", "onAIImplementStart")
-self.vehicle:requestActionEventUpdate()

Reply:
My question really was if that raiseStateChange is called on the server (is it?) then doesn't that send that event to the clients anyway to lower the implements, so is there really a need for the AIDriver doing that?

Reply:
not sure, but if look at: https://gdn.giants-software.com/documentation_scripting_fs19.php?version=script&category=70&class=10462#startAIVehicle165101 in startAIVehicle
there it seems like they get called on both server and client

Reply:
should fix these: #4820 and #5558